float64 tdoa_1
float64 tdoa_2
bool    target
